src/data/translation.ts

```diff
@@ -53,14 +53,16 @@ export const getHassTranslations = async (
   language: string,
   category: TranslationCategory,
   integration?: string,
-  config_flow?: boolean
+  config_flow?: boolean,
+  integrations?: string[]
```

Comment:
I wasn't sure if it was better to add a new arg, or replace all the existing usage?

Reply:
Let's make `integration` be string | string[]

Reply:
Yeah.  Thats much better.. Makes this tiny